plugins/ipam/dhcp/daemon.go

```diff
@@ -70,11 +71,12 @@ func (d *DHCP) Allocate(args *skel.CmdArgs, result *types.Result) error {
 
 	d.setLease(args.ContainerID, conf.Name, l)
 
-	result.IP4 = &types.IPConfig{
-		IP:      *ipn,
+	result.IPs = []*current.IPConfig{{
+		Version: "4",
```

Comment:
@AlexeyKasatkin nothing about the Allocate() call and the lease acquisition process changes.  The DHCP plugin does not currently support DHCPv6 at all, and thus it's still limited to DHCPv4.  Any invocation of the DHCP plugin will inherently try only IPv4.

But IPv6 patches would be gratefully accepted!